README.md

```diff
@@ -123,10 +123,10 @@ setx GOPATH %USERPROFILE%\go
 If you haven't defined the PATH, the command below will fail silently, and
 running `docker-credential-ecr-login` will output: `command not found`
 
-You can install this via `go get` with:
+You can install this via `go install` with:
 
 ```
-go get -u github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login
+go install github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login@latest
```

Comment:
This command doesn't work for go < 1.16. Our documentation recommends 1.12 or higher so can we please split this into the old command for go < 1.16 and this command for go >=1.16?

Reply:
@Kern-- good catch, I've added that in 3f58d83f90a06745850812617c49224d1c7e13bb

Let me know if that language looks good.

Reply:
LGTM, thanks